vdev/src/commands/install_git_hooks.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::{Result, Ok};
+use std::process::Command;
+use crate::git;
+use crate::app::CommandExt as _;
+// use std::{env, path::PathBuf};
```

Comment:
```suggestion
use crate::app::CommandExt as _;
```

If this isn't used it should be removed completely, rather than commented out.